src/installer/pkg/projects/netcoreappRIDs.props

```diff
@@ -30,6 +30,9 @@
     <OfficialBuildRID Include="linux-arm64">
       <Platform>arm64</Platform>
     </OfficialBuildRID>
+    <OfficialBuildRID Include="linux-riscv64">
```

Comment:
linux-riscv64 is an unofficial RID (i.e. there are no Microsoft builds). It is already defined below along with linux-musl-riscv64.